### PR TITLE
NMS-12641: Implement deprecated OSPF attrs and silence warnings

### PR DESCRIPTION
**Pull request by fooker**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12641

### CODE REVIEW

File: features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bgp/packets/pathattr/ExtendedCommunities.java [hunk 1]
```diff
@@ -274,16 +249,54 @@ private static Value parseOpaque(final ByteBuf buffer,
                 }
 
                 default:
-                    BmpParser.LOG.warn("Unknown Extended Community Attribute: Opaque: {}", lowType);
+                    BmpParser.RATE_LIMITED_LOG.debug("Unknown Extended Community Attribute: Opaque: {}", lowType);
                     return null;
             }
         }
 
+        private static Value parseOspfRouteType(ByteBuf buffer) {
```
Comment (christianpape) [inline features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bgp/packets/pathattr/ExtendedCommunities.java:257]: final?

File: features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bmp/TLV.java [hunk 2]
```diff
@@ -58,7 +58,7 @@ public TLV(final ByteBuf buffer, final IntFunction<T> typer, final P parameter,
         if (this.type != null) {
             this.value = this.type.parse(slice(buffer, this.length), parameter, peerInfo);
         } else {
-            BmpParser.LOG.debug("Unknown type: {}", type);
+            BmpParser.RATE_LIMITED_LOG.debug("Unknown type: {}", type);
```
Comment (christianpape) [inline features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bmp/TLV.java:61]: type is null here, right?

Comment (fooker) [reply features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bmp/TLV.java:61]: No it's not. `this.type` vs `type`.

File: features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bgp/packets/pathattr/ExtendedCommunities.java (hunk 1 above)
Comment (fooker) [reply features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bgp/packets/pathattr/ExtendedCommunities.java]: Go away!

Comment (christianpape) [reply features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bgp/packets/pathattr/ExtendedCommunities.java]: Mimimi

File: features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bmp/TLV.java (hunk 2 above)
Comment (christianpape) [reply features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bmp/TLV.java:61]: Yes. You are sooo right.

**Pull request merged**